src/Phinx/Db/Action/AddColumn.php

```diff
@@ -40,7 +42,7 @@ public function __construct(Table $table, Column $column)
      * @param array<string, mixed> $options The column options
      * @return static
      */
-    public static function build(Table $table, string $columnName, $type = null, array $options = [])
+    public static function build(Table $table, string $columnName, string|Literal $type, array $options = []): static
```

Comment:
This appears to default to null only for a few tests which were omitting a type. This doesn't seem to be valid as the phinx code checks for either Literal or string. Updated tests to specify a type.